evaluator/autoware_planning_evaluator/include/autoware/planning_evaluator/metrics/trajectory_metrics.hpp

```diff
@@ -37,6 +40,15 @@ using autoware_planning_msgs::msg::TrajectoryPoint;
 Accumulator<double> calcTrajectoryRelativeAngle(
   const Trajectory & traj, const double min_dist_threshold);
 
+/**
+ * @brief calculate large relative angle metric (angle between successive points)
+ * @param [in] traj input trajectory
+ * @param [in] vehicle_length_m input vehicle length
+ * @return calculated statistics
+ */
+Accumulator<double> calcTrajectoryResampledRelativeAngle(
+  const Trajectory & traj, const double & vehicle_length_m);
```

Comment:
(minor) primitive types do not need to be passed by reference.
```suggestion
  const Trajectory & traj, const double vehicle_length_m);
```

Reply:
Thank you very much!
I fixed it https://github.com/autowarefoundation/autoware.universe/pull/10020/commits/39efe0a523ad911a7b093fd754fcd8b95ef329a2.